code/modules/research/rdconsole.dm

```diff
@@ -437,6 +437,11 @@ proc/CallMaterialName(ID)
 									new_item.loc = L
 									L.name += " ([new_item.name])"
 									L.origin_tech = new_item.origin_tech
+									L.req_access = being_built.access_requirement
+									var/lockbox_access = null
+									for(var/A in L.req_access)
+										lockbox_access = get_access_desc(A)

```

Comment:
You could have a for loop add in the access descriptions to a string that starts off as `""`, rather than directly set the access description.